docs/BUILD

```diff
@@ -43,6 +43,36 @@ filegroup(
     srcs = glob(["root/**/*.pb"]),
 )
 
+genrule(
+    name = "api_rst",
+    srcs = ["@envoy_api//:v3_protos"],
+    outs = ["api_rst.tar"],
+    cmd = """
+    OUTDIR=$$(mktemp -d) \
+    && XDS_PROTO_SRCS=$$(cat $(location :xds_proto_srcs) | cut -d/ -f3- | tr ':' '/') \
+    && V3_PROTO_SRCS=$$(cat $(location :v3_proto_srcs) | cut -d/ -f3- | tr ':' '/') \
+    && $(location @com_google_protobuf//:protoc) \
```

Comment:
Is there a way to get further parallelism here? One of the issues I see is we're handing everything to a single worker.

Reply:
yep, the ideal is it runs `n` processes in total for number of procs

we could achieve this either by dividing the jobs up and using xargs or with python multiproc

Reply:
perhaps with bazel, but not sure how to tell bazel how many cores there are (cf https://github.com/bazelbuild/bazel/issues/10443)

Reply:
parallelizer added